core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java

```diff
@@ -427,6 +427,10 @@ private void runInternal() {
                 } else {
                   mResponse.onNext(response);
                 }
+                if (mContext.getBlockReader() == null) {
+                  mContext.setCounter(MetricsSystem.counter(
```

Comment:
Where you update the value of this metric?

Reply:
Thanks for your review. 
Only when read from alluxio cache data, `mContext.getBlockReader()` will is null. 
Updating the value using common code as follows:
https://github.com/Alluxio/alluxio/blob/221f0ed5851417ffc6cae4758d112d490e25f930/core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java#L550-L568.